external/ag-website-shared/src/content/license-features/gridFeaturesMatrix.json

```
@@ -1,4 +1,29 @@
 [
+    {
+        "group": {
+            "name": "AI Features"
+        },
+        "items": [
+            {
+                "label": {
+                    "name": "MCP Server",
+                    "link": "https://www.ag-grid.com/react-data-grid/mcp-server/"
```

Should we do `/data-grid/` here for the redirects?

```suggestion
                    "link": "https://www.ag-grid.com/data-grid/mcp-server/"
```

---

Good idea Going to leave to post release.